en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,113 @@
+  Populate it with the information from the [Github project board](https://github.com/silverstripe-security/security-issues/projects/1).
+* Link to silverstripe.org security release page in the changelog.
+* Update the _draft security advisory_ on GitHub.
+* Move the issue to "Awaiting Release" in the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+ 
+### After release
+
+* Publish silverstripe.org security release page
+* Respond to issue reporter with reference to the release on the same discussion thread (cc security@silverstripe.org)
+* Publish the security advisory on GitHub. This will also publish the CVE.
+* Open a pull request to [https://github.com/FriendsOfPHP/security-advisories/tree/master/silverstripe](https://github.com/FriendsOfPHP/security-advisories/tree/master/silverstripe) repository.
+* Move the issue to "Done" in the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+
+### Detailed CVE and CVSS Guidance
+
+* In the GitHub _draft security advisory_, we follow certain conventions on fields:
+  * **Affected product** should have one entry for each composer package affected by the vulnerability. If the patch is backported to older release lines, there should be entries for those older releases.
+    * **Ecosystem** should always be `composer`.
+    * **Package name** sholud match the package name in Packagist.
+    * **Affected versions** should be a string reprenting a range of version affected by the vulnerability.
+    * **Patched versions** should be a list version where the vulnerability was/will be patched.
+  * **Severity** should be set to *Assess severity using CVSS* and the CVSS option should be specified in the calculator.
+  * **Common weakness enumerator (CWE)** should be left blank if no suitable option can be found in the list.
+  * **CVE identifier** should be set to "Request CWE ID later".
+  * **Title** and **Description** should be descriptive enough that people can evaluate the risk without being so specific that would-be-hackers can start exploiting them right away with no efforts. 
+
+These settings can be changed later, so you can iterate over them.
+
+## Special circumstances
+
+### An issue has been reported publicly on GitHub
+
+Occasionally, some developers will open a public issue on GitHub reporting a vulnerability, either because they are unaware of the proper process or because they did not immediately grasp the security implication of the bug being reported.
+
+In those situations, the security team will assess how likely it was that the vulnerability was widely noticed based on how long the issue was opened on GitHub and how explicit the description was. If the risk of the vulnerability having been noticed is small, an owner on the Silverstripe GitHub organisation will delete the public facing issue and create a private one instead.
```

Comment:
```suggestion
In those situations, the security team will assess how likely it was that the vulnerability was widely noticed based on how long the issue was opened on GitHub, whether there were any interactions from other GitHub users, and how explicit the description was. If the risk of the vulnerability having been noticed is small, an owner on the Silverstripe GitHub organisation will delete the public facing issue and create a private one instead.
```
You've got information about what happens if the risk is small, but what if it is _not_ small?

Reply:
I've added a blurb.

Reply:
The suggested change above wasn't made yet

Reply:
I've added the blurb.